internal/provider/features_test.go

```diff
@@ -1237,7 +1256,7 @@ func TestExpandFeaturesSubscription(t *testing.T) {
 			},
 		},
 		{
-			Name: "No Downtime Resize Enabled",
+			Name: "Subscription cancellation on destroy is Disabled",
```

Comment:
I also fixed the test name which was not edited after copy-paste. This change is added as a separate commit, but I can also remove it completely